test/libsolidity/smtCheckerTests/special/msg_sender_3.sol

```diff
@@ -0,0 +1,21 @@
+pragma experimental SMTChecker;
+
+contract C {
+	address owner;
+
+	constructor() {
+		owner = msg.sender;
+		assert(owner >= address(0)); // should hold
+	}
+}
+
+contract D {
+	address owner;
+
+	constructor() {
+		unchecked {
+			owner = msg.sender;
+			assert(owner >= address(0)); // should hold
```

Comment:
Is the comment here valid since it is inside an unchecked block that is presumably unchecked?

Reply:
`assert` is still checked and the type constraints are valid also in unchecked blocks.